### DISCUSSION TITLE
@Ip 在docker部署环境中，无法获取到真实的ip

### DISCUSSION THREAD

**Discussion by ywachao** [nestcn/docs.nestjs.cn#270] [2-Q&A]
同样的docker环境，我自己写中间件，是可以获取到真实的访问ip，但是用@Ip 只能获取到Nginx容器的内网ip

**Comment by ganshiqingyuan** [comment]
x-forward-for 在nginx里设置一下

**Reply by AbsenceSliet** [reply]
棒棒的

**Comment by zuohuadong** [comment]
可以参考：
https://segmentfault.com/a/1190000016141653
搜索 nginx 反向代理 获取真实IP 。
